main.tf

```diff
@@ -52,3 +55,25 @@ module "eks_node_group" {
   eks_cluster_endpoint = aws_eks_cluster.eks.endpoint
   eks_cluster_ca = aws_eks_cluster.eks.certificate_authority.0.data
 }
+
+module "karpenter" {
+  source = "terraform-aws-modules/eks/aws//modules/karpenter"
+  version = "20.8.5"
+
+  cluster_name = aws_eks_cluster.eks.id
+  node_iam_role_name = "KarpenterTF-${aws_eks_cluster.eks.id}"
+
+  # Attach additional IAM policies to the Karpenter node IAM role
+  node_iam_role_additional_policies = merge({
+    AmazonSSMManagedInstanceCore = "arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore"
```

Comment:
is it required by karpenter to work? (`AmazonSSMManagedInstanceCore`)
I saw it was in the example on their repo, but I don't think its a must
do we need it?